components/templates.templ

```diff
@@ -60,3 +60,31 @@ templ Template(template models.Template, answers []models.Answer) {
 		}
 	</div>
 }
+
+templ GetStarted() {
+	<h2>Get Started with tracking your goals</h2>
+	<form hx-post="/template-questionnaire" hx-swap="innerHTML">
+		<div>
+			<input type="checkbox" id="fitness" name="options" value="fitness"/>
```

Comment:
I think we'll make these radio buttons and only be able to make one type of goal each time